### DISCUSSION TITLE
How to remove consent with Sharepoint Business account?

### DISCUSSION THREAD

**Discussion by buergi** [abraunegg/onedrive#2717] [Q&A]
I like to make to sync a Sharepoint read only which I already had configured and synced with write permissions. After setting `read_only_auth_scope="true"`, I keep getting the request to remove my existing application access consent via https://account.live.com/consent/Manage .
However, when I try to log in on this site, after entering my mail address I keep getting the message "This Microsoft account doesn't exist" although it does exist and I can log in with the identical address on Sharepoint, Outlook, etc. I assume it has something to do with the fact that it is a business account. Any idea how I can manage my consent?

**Comment by abraunegg** [comment]
@buergi 
If your account is not visible under that portal or says its not a valid account - not much I can do to assist here.

Given this is a Business account, and this comes with Microsoft support - potentially raise a support request to Microsoft as to why your account is not able to be used in the consent portal?
